persistence/soft-index/src/main/java/org/infinispan/persistence/sifs/configuration/IndexConfiguration.java

```diff
@@ -0,0 +1,86 @@
+package org.infinispan.persistence.sifs.configuration;
+
+import static org.infinispan.persistence.sifs.configuration.Element.INDEX;
+
+import org.infinispan.commons.configuration.ConfigurationInfo;
+import org.infinispan.commons.configuration.attributes.AttributeDefinition;
+import org.infinispan.commons.configuration.attributes.AttributeSet;
+import org.infinispan.commons.configuration.elements.DefaultElementDefinition;
+import org.infinispan.commons.configuration.elements.ElementDefinition;
+
+public class IndexConfiguration implements ConfigurationInfo {
+
+   public static final AttributeDefinition<String> INDEX_LOCATION = AttributeDefinition.builder("indexLocation", "Infinispan-SoftIndexFileStore-Index").immutable().autoPersist(false).xmlName("path").build();
```

Comment:
IMO the data and index paths should be together (or even better have a single path). Otherwise this is one place where a separate element looks justified even to me :)

Reply:
Although it's possible to change the schema, I'm not really looking forward to it :smile:, as we'd need to change the XML parsing to accept both formats (the 9.x one and the 10.x one) and the user would need to change the config when migrating.

Reply:
I was merely reflecting the current state of affairs, could you update https://issues.jboss.org/browse/ISPN-10545 whether we should change it or not?